account-kit/signer/src/client/index.ts

```diff
@@ -518,6 +518,11 @@ export class AlchemySignerWebClient extends BaseSignerClient<ExportWalletParams>
     } = args;
     const { codeChallenge, requestKey, authProviders } =
       await this.getOauthConfigForMode(mode);
+    if (!authProviders) {
+      throw new Error(
+        "No auth providers found, please enable auth providers on the alchemy dashboard here: https://dashboard.alchemy.com/accounts"
+      );
```

Comment:
nit: can we add this to `errors.ts` as a new error type? you can set the docsPath to be `/react/getting-started`